### PR TITLE
Ansible: Removed the install DKMS steps, and updated the Install NVid…

### PR DESCRIPTION
**Pull request by CJKwork**
…ia CUDA toolkit step to install cuda-toolkit-9-0

Fix for issue https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/467

### CODE REVIEW

**Comment by sxa555** [comment]
I worked with Clive to adjust the commit message :-)

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml [hunk 1]
```diff
@@ -37,27 +37,6 @@
...
-  when:
-    - cuda_installed.stat.isdir is not defined
-    - ansible_distribution == "RedHat"
-    - ansible_distribution_major_version == "7"
-    - ansible_architecture == "ppc64le"
-  tags: nvidia_cuda_toolkit
-
 - name: Download NVidia CUDA Repo RPM - RHEL 7 on PPC64LE
   get_url:
     url: https://developer.download.nvidia.com/compute/cuda/repos/rhel7/ppc64le/cuda-repo-rhel7-9.0.176-1.ppc64le.rpm
```
Comment (jdekonin) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml]: Walking through the issue to #467, I see @keithc-ca mentions v9.2.148 where this installs v9.0.176.  @keithc-ca is there an OpenJ9 support statement for the version of cuda on PPC64LE?

**Comment by sxa555** [comment]
Cheers for checking up on that one @jdekonin :-)

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml (hunk 1 above)
Comment (keithc-ca) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml]: We will need to update to 9.2 to get a version that is compatible with gcc 7. CUDA 9.1 `crt/host_config.h` says:
```c
#if defined(__GNUC__)
#if __GNUC__ > 6
#error -- unsupported GNU version! gcc versions later than 6 are not supported!
```
Older versions have even lower limits on the version of gcc.

**Pull request merged**